src/api/app/policies/user_policy.rb

```diff
@@ -0,0 +1,9 @@
+class UserPolicy < ApplicationPolicy
+  def update?
+    @user.can_modify_user?(@record)
+  end
+
+  def show?
+    @user == @record || @user.is_admin?
```

Comment:
Isn't that the same as ``update?``?

Reply:
True. I could re-use the can_modify method.